### ISSUE TITLE
Tabs disappears when switch fragments

### ISSUE THREAD

**Issue by darenpaten** [gilgoldzweig/EasyTabs#8]
Hello.
I have been using EasyTabs for quite a while now and it truly is a great work. But recently I have used it in a fragment, but the problem is that whenever I switch to other fragments and then come back to the fragment containing the tabs, the tabs just disappears along with whatever were in them. It is just blank. What can I do to resolve this? Please respond as soon as possible.
Thank you!!

**Comment by gilgoldzweig** [comment]
re we talking about fragments inside the tabs?

**Comment by darenpaten** [comment]
No, fragments of a navigation drawer. Whenever I switch to other fragments in the navigation drawer and then come back to the one containing the tabs, the fragments in the tabs disappears.

**Comment by gilgoldzweig** [comment]
So you're not inflating the parent of the easytabs view properly

**Comment by darenpaten** [comment]
I was just using the default code that is generated by Android Studio for the Navigation Drawer. Do you have any idea for how I can fix this?

**Comment by gilgoldzweig** [comment]
Don't ever use the generated code it's bad read online on how to build it
and do as follow

**Comment by lucasharada** [comment]
@darenpaten Any solution for this? I'm having the same issue.

**Comment by Vuyiswamb** [comment]
i solved the Problem today through the help of the post https://stackoverflow.com/questions/27142850/tab-content-disappeared-after-change-page 
`
use ChildFragmentManager 

  adapter = new CustomPagerAdapter(**ChildFragmentManager**, fragmentList, titles);`
